protos/sources/data-node/api/v2/trading_data.proto

```diff
@@ -799,12 +799,12 @@ service TradingDataService {
     option (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_operation) = {tags: "Teams"};
   }
 
-  // Get referral fee statistics
+  // Get fees statistics
```

Comment:
Is the description for this one still correct? I'd also expect stats about fees to tell me about the fees paid (which isn't mentioned) and discounts, but not necessarily about rewards.

Reply:
It seems then it should say “rewards based on fees paid or earned” since it sounds like it won’t show all rewards then, right?